1.1/terms_and_defns.ttl

```diff
@@ -0,0 +1,159 @@
+@prefix cs: <http://www.opengis.net/def/geosparql/td> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix td: <http://www.opengis.net/def/geosparql/td/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+cs:
+    a owl:Ontology , skos:ConceptScheme ;
+    dcterms:created "2021-06-22"^^xsd:date ;
+	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
+    dcterms:modified "2021-06-22"^^xsd:date ;
+    dcterms:provenance "Each term (Concept) listed here indicates its own provenance since they are derived from other vocabularies and non-vocabulary sources."@en ;
+	dcterms:publisher [
+		a sdo:Organization ;
+		sdo:name "Open Geospatial Consortium" ;
+		sdo:url "https://www.ogc.org"^^xsd:anyURI ;
+	] ;
+    owl:versionInfo "OGC GeoSPARQL 1.1" ;
+    skos:definition "A vocabulary (taxonomy) of the Terms and Definitions listed by the GeoSPARQL 1.1 standard. These terms appear as a vocabulary resource (this file) and are also listed in Clause 4 of the GeoSPARQL 1.1 Specification"@en ;
+    skos:hasTopConcept
+        td:Subject ,
+        td:Predicate ,
+        td:Object ,
+        td:Resource ,
+        td:Class ,
+        td:Property ,
+        td:ObjectProperty ;
+        # td:DatatypeProperty ,
+        # td:AnnotationProperty 
+    skos:prefLabel "GeoSPARQL Terms & Definitions" ;
+.
+
+# Concept Template
+# td:XXX a skos:Concept ;
+#     dcterms:provenance ""@en ;
```

Comment:
The way `dcterms:provenance` is used now in the concept scheme seems to have more to do with the source. So let's use `dcterms:source` for that? And use `dcterms:provenance` for documenting changes (which at the moment is not required because the concept scheme is fresh).

Reply:
> So let's use `dcterms:source` for that?

I used to do that but note this guidance from [Dublin Core `source`](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/source): "This property is intended to be used with non-literal values. The described resource may be derived from the related resource in whole or in part. Best practice is to identify the related resource by means of a URI or a string conforming to a formal identification system.".

So lots of people use `dcterms:source` not as intended, it's an ObjectProperty, not an AnnotationProperty.

So what to use for annotations? `dcterms;provenance`? Happy to consider options other than `dcterms:source`.

Reply:
If I look at the examples, it seems that the annotation literals can be replaced by IRIs. For example, in 
`td:Property dcterms:provenance "From the RDF Schema vocabulary (RDFS)"@en `, couldn't the IRI of the RDF Schema vocabulary be used? I think we should be safe to assume that all definition sources are available online somehow, with a persistent IRI.